cmd/talosctl/cmd/talos/image.go

```diff
@@ -153,6 +156,87 @@ var imageDefaultCmd = &cobra.Command{
 	},
 }
 
+// imageIntegrationCmd represents the integration image command.
```

Comment:
not sure why we need this in `talosctl` doesn't the list from conformance container image not enough?

Reply:
Not really, because we need to do deduplication, also need images from the `default` command, but not all of them, and also `installer` we have to format with current TAG rather than stored one. All of this could be done using bash script, but I think that simple Go code is better than complex bash script.